src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final int index = random.nextInt(Color.values().length);
+
+    public String getRandomColor() {
+        return Color.values()[index].name();
+    }
```

Comment:
In such cases, you always have the same color
```suggestion

    public String getRandomColor() {
        return Color.values()[random.nextInt(Color.values().length)].name();
    }
```